identity/app/views/profile/profilingConsent.scala.html

```diff
@@ -0,0 +1,29 @@
+@import views.support.fragment.ConsentChannel._
+
+@(
+    idUrlBuilder: services.IdentityUrlBuilder,
+    idRequest: services.IdentityRequest,
+    privacyForm: Form[_root_.form.PrivacyFormData],
+    user: com.gu.identity.model.User,
+)(implicit request: RequestHeader, messages: play.api.i18n.Messages)
+<form method="post" action="@idUrlBuilder.buildUrl("/privacy/edit", idRequest)" novalidate>
+    @views.html.helper.CSRF.formField
+    <div class="manage-account__switches manage-account__switches--single-column js-manage-account__check-allCheckbox__ignore">
+        <ul>
+        @helper.repeatWithIndex(privacyForm("consents"), min=1) { (consentField, index) =>
+            @if(isProfilingChannel(consentField, user)) {
+                <p class="identity-title-explainer">From time to time we may use your personal data for marketing analysis. That includes looking at what products or services you have bought from us and what pages you have been viewing on
+                    guardian.com and other Guardian websites (e.g. Guardian Jobs or Guardian Holidays). We do this to understand your interests and preferences so that we can make out marketing communication more relevant to you.</p>
```

Comment:
should this all be on the same line? also I think we'll have to put `theguardian.com` as we don't own `guardian.com`